src/components/tab-bar/tab-bar.tsx

```diff
@@ -72,7 +71,7 @@ export class TabBar {
     }
 
     private setup() {
-        const element = this.element.shadowRoot.querySelector('.mdc-tab-bar');
+        const element = this.host.shadowRoot.querySelector('.mdc-tab-bar');
```

Comment:
We use `host` in `tab-panel` and I think it's less confusing to not have both `element` and `this.element`.